### PR TITLE
Make Login Button More Noticeable

### PR DESCRIPTION
**Pull request by sigmachirality**
The Login Button is styled using CLUBS_RED, so it should change color when the colors are switched over.

Screenshot from 2020-01-21 14-31-29
Screenshot from 2020-01-21 14-31-46

### CODE REVIEW

File: frontend/components/Header/Links.js [hunk 1]
```diff
@@ -4,14 +4,39 @@ import s from 'styled-components'
...
   opacity: 0.5;
   margin-right: 4px;
 `
 
+const LoginButton = s.a`
+  border: 0;
+  background-color: ${CLUBS_RED};
+  padding: 14px 20px;
+  margin: auto;
+  color: ${SNOW} !important;
```
Comment (cacabo) [inline frontend/components/Header/Links.js]: Would prefer `WHITE_ALPHA(0.8)` or just `WHITE`

File: frontend/components/Header/Links.js [hunk 2]
```diff
@@ -4,14 +4,39 @@ import s from 'styled-components'
...
+
+  &:hover {
+    color: ${SNOW_ALPHA} !important;
```
Comment (cacabo) [inline frontend/components/Header/Links.js]: Generally want to keep font color the same but then make the background color a slightly darker version. So feel free to make a color called CLUBS_RED_DARK or CLUBS_RED_DEEP. A good way to do this is to google the hex number then click a little bit to the bottom right of the color in the color picker until there is a noticeable but not extreme difference between the two colors. 

lmk if you need help with this.

File: frontend/components/Header/Links.js [hunk 3]
```diff
@@ -4,14 +4,39 @@ import s from 'styled-components'
...
+  }
+
+  ${mediaMaxWidth(MD)} {
+    padding: 8px 0;
+    padding-top: .4rem;
```
Comment (cacabo) [inline frontend/components/Header/Links.js]:
```suggestion
    padding-top: 0.4rem;
```

File: frontend/components/Header/Links.js [hunk 4]
```diff
@@ -4,14 +4,39 @@ import s from 'styled-components'
...
+    width: 5rem !important;
```
Comment (cacabo) [inline frontend/components/Header/Links.js]: Why is fixing the width necessary? Can't we just use wider padding on the sides?

File: frontend/components/Header/Links.js (hunk 1 above)
Comment (sigmachirality) [reply frontend/components/Header/Links.js]: Done.

File: frontend/components/Header/Links.js (hunk 2 above)
Comment (sigmachirality) [reply frontend/components/Header/Links.js]: Done, though check out the hex number I chose and make sure I didn't go too far with the difference.

File: frontend/components/Header/Links.js [hunk 5]
```diff
@@ -4,25 +4,57 @@ import s from 'styled-components'
...
   opacity: 0.5;
   margin-right: 4px;
 `
 
+const LoginButton = s.a`
+  border: 0;
+  background-color: ${CLUBS_RED};
+  padding: 14px 20px;
+  margin: auto;
+  color: ${WHITE_ALPHA(0.8)} !important;
```
Comment (cacabo) [inline frontend/components/Header/Links.js]:
```suggestion
  color: ${WHITE_ALPHA(0.8)} !important;
  transition: color ${SHORT_ANIMATION_DURATION}ms ease,
              background ${SHORT_ANIMATION_DURATION}ms ease;
```

Comment (sigmachirality) [reply frontend/components/Header/Links.js]: lgtm!

**Pull request merged**